packages/timeline-state-resolver/src/devices/deviceContainer.ts

```diff
@@ -11,8 +11,8 @@ export { DeviceContainerEvents }
  * names and id's) to prevent a costly round trip over IPC.
  */
 export class DeviceContainer<TOptions extends DeviceOptionsBase<any>> extends BaseRemoteDeviceIntegration<TOptions> {
-	protected _device: ThreadedClass<Device<TOptions>>
-	public onChildClose: () => void | undefined
+	protected _device!: ThreadedClass<Device<TOptions>>
```

Comment:
Perhaps we should instead add this to as an argument to the construtor, and in `static create()`:

```typescript
const device = await threadedClass<Device<TOptions>, TCtor>(
	orgModule,
	orgClassExport,
	[deviceId, deviceOptions, getCurrentTime] as any, // TODO types
	threadConfig
)
const container = new DeviceContainer(device, deviceOptions, threadConfig)
```